server/api.js

```diff
@@ -8,5 +8,10 @@ router.get("/", (_, res) => {
 	logger.debug("Welcoming everyone...");
 	res.json({ message: "Hello, world!" });
 });
+let rhythms = require("./data/rhythmsData.json");
```

Comment:
Looking at this again, we are mixing two ways of bringing code into this file - `import` and `require`.

I think it's fine since we're doing temporary JSON, but good info for you to know: https://stackoverflow.com/questions/7163061/is-there-a-require-for-json-in-node-js